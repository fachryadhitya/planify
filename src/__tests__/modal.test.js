import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Modal from '../components/modal/modal';


describe('Modal Component', () => {
  it('Render successfully', () => {
    const { container } = render(<Modal isShowing />);
    expect(container).toMatchSnapshot();
  });

  it('Shows form correctly', () => {
    const { getByTestId } = render(<Modal isShowing />);
    expect(getByTestId('form')).toBeInTheDocument();
  });

  // it ('save form correctly', () => {
  //   const button = screen.getByText("Save");
  //   expect(button).toBeInTheDocument();
  // })

  //   it('Render component with specific width', () => {
  //     const { getByTestId, getByText } = render(<AdsBanner width={700} height={700} />);
  //     expect(getByTestId('ads-banner')).toHaveStyle('width: 700px');
  //     expect(getByText('700 x 700')).toBeInTheDocument();

  //   });
});
