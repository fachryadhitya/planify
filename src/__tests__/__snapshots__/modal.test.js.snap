// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Component Render successfully 1`] = `<div />`;
